<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionLoad_Objects"/>
    <addaction name="actionSave_Objects"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionGenerate_Grid"/>
    <addaction name="separator"/>
    <addaction name="actionMark_Wire"/>
    <addaction name="actionMark_Via"/>
    <addaction name="actionMark_Insulator"/>
    <addaction name="actionSelect"/>
    <addaction name="actionClear_All"/>
    <addaction name="actionSet_Param"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Train"/>
    <addaction name="actionExtract"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Wire"/>
    <addaction name="actionShow_Via"/>
    <addaction name="actionShow_Wire_Edge"/>
    <addaction name="actionShow_Via_Edge"/>
    <addaction name="actionShow_Via_Wire_Edge"/>
    <addaction name="actionShow_Select"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionGenerate_Grid">
   <property name="text">
    <string>Generate Grid</string>
   </property>
  </action>
  <action name="actionMark_Wire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mark Wire</string>
   </property>
  </action>
  <action name="actionMark_Via">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mark Via</string>
   </property>
  </action>
  <action name="actionMark_Insulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mark Learn Area</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionLoad_Objects">
   <property name="text">
    <string>Load Objects</string>
   </property>
  </action>
  <action name="actionSave_Objects">
   <property name="text">
    <string>Save Objects</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionShow_Wire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Wire</string>
   </property>
  </action>
  <action name="actionShow_Via">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Via</string>
   </property>
  </action>
  <action name="actionShow_Wire_Edge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Wire Edge</string>
   </property>
  </action>
  <action name="actionShow_Via_Edge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Via Edge</string>
   </property>
  </action>
  <action name="actionShow_Via_Wire_Edge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Via Wire Edge</string>
   </property>
  </action>
  <action name="actionStart_Train">
   <property name="text">
    <string>Start Train</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionShow_Select">
   <property name="text">
    <string>Show Select...</string>
   </property>
  </action>
  <action name="actionSet_Param">
   <property name="text">
    <string>Set Param</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>Extract</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
